JsonSchema.ArrayExt/JsonSchema.ArrayExt.csproj

```diff
@@ -42,6 +44,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
+		<PackageReference Include="PolySharp" Version="1.14.1">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="JetBrains.Annotations" Version="2021.2.0" />
```

Comment:
The Jetbrains should be inherited from JsonSchema.  Maybe you copied an extra line?

Reply:
Oops.